.github/workflows/docker-release.yml

```diff
@@ -1,27 +1,27 @@
-name: Daily build
+# See RELEASING.md#DockerImage for more details about the steps in this workflow.
+name: Daily Release AtlasCLI Docker Image
 on:
   schedule:
     - cron: "0 1 * * *" # Every day at 1:00 AM
-  workflow_dispatch:
+  workflow_dispatch: # Run the action manually
 jobs:
-  release_image:
-    name: Build and publish docker image
+  build_images:
+    name: Build and publish docker image to staging registry
     runs-on: ubuntu-latest
     env:
-      IMAGE_REPOSITORY: mongodb/atlas
+      STAGING_IMAGE_REPOSITORY: mongodb/apix_test
       PLATFORMS: linux/amd64,linux/arm64
-      QUAY: quay.io
     steps:
       - name: Check out code
-        uses: actions/checkout@v4
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
```

Comment:
[q] why do we need to pin the version to commit shas across this file?

Reply:
This is a good article to get more info: https://www.paloaltonetworks.com/blog/prisma-cloud/github-actions-worm-dependencies/

TLDR: 

> Pin actions using a commit hash to reduce the risk of using a maliciously modified action.